apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/DMEClaimTransformerV2Test.java

```diff
@@ -43,17 +46,30 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
 
 /** Unit tests for {@link DMEClaimTransformerV2}. */
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
 public final class DMEClaimTransformerV2Test {
   /** The parsed claim used to generate the EOB and for validating with. */
   DMEClaim claim;
   /** The EOB under test created from the {@link #claim}. */
   ExplanationOfBenefit eob;
+  /** The transformer under test. */
+  ClaimTransformerInterfaceV2 claimTransformerInterface;
   /** The fhir context for parsing the test file. */
   private static final FhirContext fhirContext = FhirContext.forR4();
-  /** The transformer under test. */
-  DMEClaimTransformerV2 dmeClaimTransformer;
+  /** The mock metric registry. */
+  @Mock MetricRegistry mockMetricRegistry;
+  /** The mock metric timer. */
+  @Mock Timer mockTimer;
+  /** The mock metric timer context (used to stop the metric). */
+  @Mock Timer.Context mockTimerContext;
```

Comment:
Are there any assertions being performed on these mocks?  If not the original idea of just passing in a real throw away registry would be simpler.

Reply:
IMO better for the understanding of the testing methodology to have the mock even if unused; besides its hardly any different having a single mock line here or creating an empty registry.

Reply:
I don't see how adding unused fields and creating mocks that are not tested adds anything but time and cognitive load.  Better to use concrete objects unless mocks are needed.  Reduces the amount of code to read when understanding what the tests are doing.

Also it's not a single mock line.  There are the Timer and TimerContext mocks as well and the statements to trigger them being returned.

For that matter those are the only mocks in the class and they are not being used for anything so there are the annotations on the class that aren't necessary as well.  We should either `verify()` that the mocks were invoked or delete them.

Reply:
This gets a bit into the testing pattern that I'm trying to instill here.

The final result would likely be making a single dummy object declared at the test level that gets passed into each instance, which will be the same 'complexity' as making a mock object for each external dependency (lest you repeat creating the same object in each constructor). By having all mocks it reinforces the testing strategy (in code) of having  external dependencies be mocked, which _reduces_ the complexity of understanding the tests by making a consistent pattern to follow in other unit tests.